runtime/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = '2018'
 name = 'joystream-node-runtime'
 # Follow convention: https://github.com/Joystream/substrate-runtime-joystream/issues/1
 # {Authoring}.{Spec}.{Impl} of the RuntimeVersion
-version = '6.12.0'
+version = '6.12.2'
```

Comment:
When updating version here we should also remember to change the version numbers in [runtime/src/lib.rs](https://github.com/Joystream/joystream/blob/development/runtime/src/lib.rs#L124)

Reply:
Increased from 0 to 1.